app/Http/Controllers/PlantController.php

```diff
@@ -44,6 +44,7 @@ public function index(Plot $plot, Request $request)
             'quadrant',
             'plant_type_id',
             'plot_id',
+            'user_id',
```

Comment:
adding the table name prevents future conflict if we ever add user_id to species table. Since user_id is a common column name, I'd suggest we always prefix it with the table name.
```suggestion
            'plants.user_id as user_id',
```